app/views/view_data/services/show.html.erb

```diff
@@ -33,22 +33,16 @@
 
     <div class="column-two-thirds service-context">
 
-      <h2 class="bold-medium">What is the purpose of the service?</h2>
+      <h2 class="bold-medium">Purpose of the service</h2>
       <%= render_markdown(@service.purpose) %>
 
-      <h2 class="bold-medium">How does the service work?</h2>
+      <h2 class="bold-medium">How it works</h2>
       <%= render_markdown(@service.how_it_works) %>
 
-      <h2 class="bold-medium">Who are the typical users of the service?</h2>
+      <h2 class="bold-medium">Users</h2>
       <%= render_markdown(@service.typical_users) %>
 
-      <h2 class="bold-medium">How frequently is the service used?</h2>
```

Comment:
Are we doing away with these two heading entirely?  We can clean up some stuff if so...